workshops/collab_sketch/README.md

```diff
@@ -82,6 +82,41 @@ firebase.initializeApp(config);
 var pointsData = firebase.database().ref();
 ```
 
+Let's quickly walk through what a part of this code does.
+
+```js
+var config = {
+  (...) // stuff hidden here for simplicity
+}
+```
+
+This section of code creates a new `object` in JavaScript named `config`.  You can think of `object`s like containers for things like text or numbers which we call `properties`. 
+
+`properties` are values that we store by name in the `object` and then retrieve at a later time.
+
+```js
+var config = {
+  apiKey: "AIxaSyGsAkHke9lXEU_97a8rYpMn7gOH3eWDxrM",
+  (...)
+}
+```
+
+The first thing we've added to the object is a `property` named `apiKey`, which we assigned a jumbled value of text to.  To access this value, we can write:
```

Comment:
Should there be an explanation of what an `apiKey` is? Right now `config` and `apiKey` and `authDomain` are foreign words that make it harder to grasp why this object would have such properties. Or maybe add a line like "To configure our Firebase app, we'll need several things: an API key (which does ?allows our app to access Firebase?), the domain of our Firebase app (which ?allows Firebase to recognize our app? what does this do; I am not sure myself), and ..." etc